packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -485,150 +462,6 @@ export const prepareVaultManagerKit = (
           updateQuote(outstandingQuote, highestDebtRatio, liquidationMargin);
           trace('update quote', collateralBrand, highestDebtRatio);
         },
-
-        async processLiquidations() {
```

Comment:
It seems odd to take this out.

sure, we can put it back, but the git history will get obscured. hm.

Reply:
liquidations will be invoked in a very different way. Connecting the new code to the previous didn't seem helpful, and I couldn't identify a subset of this code that I could retain and keep lint happy